src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -260,18 +260,35 @@ public String prepareRespondentResponseText(CaseData caseData, Party respondent,
         MultiPartyScenario scenario = getMultiPartyScenario(caseData);
         String defaultText = "";
         if (scenario.equals(ONE_V_ONE) || scenario.equals(TWO_V_ONE)) {
-            switch (caseData.getRespondent1ClaimResponseType()) {
-                case COUNTER_CLAIM:
-                    defaultText = "RPA Reason: Defendant rejects and counter claims.";
-                    break;
-                case FULL_ADMISSION:
-                    defaultText = "RPA Reason: Defendant fully admits.";
-                    break;
-                case PART_ADMISSION:
-                    defaultText = "RPA Reason: Defendant partial admission.";
-                    break;
-                default:
-                    break;
+            if (SPEC_CLAIM.equals(caseData.getSuperClaimType())) {
+                //TODO cover unit test to this block
```

Comment:
Is this TODO to be done in this ticket or in the future?

Reply:
It has actually been covered in this ticket.